block_form_assem_less_rollers4:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  #Effort Controllers
  wheel1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: wheel1
    pid: {p: 9, i: 0.0, d: 1}

  wheel2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: wheel2
    pid: {p: 9, i: 0.0, d: 1}

  wheel3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: wheel3
    pid: {p: 9, i: 0.0, d: 1}
